pkg/userdata/bottlerocket/node_userdata.go

```diff
@@ -36,20 +37,42 @@ superpowered = true
 source = "{{.BootstrapContainerSource}}"
 user-data = "{{.BootstrapContainerUserData}}"
 {{- end -}}
+`
+
+	networkInitTemplate = `{{ define "networkInitSettings" -}}
+[settings.network]
+https-proxy = "{{.HTTPSProxyEndpoint}}"
+no-proxy = "{{.NoProxyEndpoints}}"
+{{- end -}}
+`
+	registryMirrorTemplate = `{{ define "registryMirrorSettings" -}}
```

Comment:
where is this one and `registryMirrorCACertSettings` being used?

Reply:
my bad, forgot to push the update earlier